### ISSUE TITLE
Image visualization for values outside the range of [0,1]

### ISSUE THREAD

**Issue by csagonas** [menpo/menpo#704]
Currently, it looks like the visualization function doesn't clip the pixels that are outside the range [0,1] and displays these pixels in a strange way.

![menpo_visualization](https://cloud.githubusercontent.com/assets/4290551/15705005/16eee8c2-27e5-11e6-8b3a-a123ad29ee2f.jpg)

It would be great if this can be configured  i.e., to automatically clip.

**Comment by patricksnape** [comment]
@nontas I guess this is a Matplotlib issue?

**Comment by nontas** [comment]
Yes this is a `matplotlib` default behaviour. This is not an issue for greyscale images, since the pixels' values are normalized by the selected colormap (e.g. `matplotlib.cm.gray`). I think that this is an issue _only_ for RGB images for which you cannot space a different colormap. 

`matplotlib.pyplot.imshow()` has a `norm` and `vmin`, `vmax` arguments which are intended to normalize the pixels, but this doesn't seem to work for RGB images. For example:

```
%matplotlib inline
import matplotlib.pyplot as plt
from matplotlib.colors import Normalize
import menpo.io as mio

im = mio.import_builtin_asset.lenna_png()
im.pixels = im.pixels + 0.2
print(im.pixels_range())

norm = Normalize(vmin=0., vmax=1., clip=True)
plt.imshow(im.rolled_channels(), norm=norm, vmin=0, vmax=1)
```
still plots
![download 2](https://cloud.githubusercontent.com/assets/4224739/15818899/d91c8ea4-2bd7-11e6-9d0f-5e24427f1924.png)

The only way to fix this issue is to normalize the image pixels right before rendering, e.g.
```
%matplotlib inline
import matplotlib.pyplot as plt
import menpo.io as mio

im = mio.import_builtin_asset.lenna_png()
im.pixels = im.pixels + 0.2
print(im.pixels_range())

img = im.rolled_channels()
plt.imshow(img / img.max())
```
which renders
![download 3](https://cloud.githubusercontent.com/assets/4224739/15818989/59c316ea-2bd8-11e6-9294-6e9f10862d54.png)

However, I think that this should **not** happen within the `visualize_appearance_model` widget, since it corrupts the actual appearance instance.

**Comment by patricksnape** [comment]
I think what @csagonas really wants is just a sensible colour rather than blue 😛 I think he would be fine with showing corrupted colours, he would just prefer to have them default as black or white rather than blue!
